src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -63,7 +63,7 @@ public function exists($name)
      *
      * @throws \Twig_Error_Loader if the template could not be found
      */
-    protected function findTemplate($template)
+    protected function findTemplate($template, $throw = true)
     {
         $logicalName = (string) $template;
```

Comment:
Indeed, that should work, but it's a bit slower as we need to parse the string to convert it again to an object.